packages/dashboard/src/components/deliveriesTable.tsx

```diff
@@ -40,6 +56,10 @@ interface TableItem {
   templateId?: string;
   templateName?: string;
   channel: ChannelType;
+  body: string | null;
```

Comment:
Yah, it does look like you've written this code to only work with email deliveries. I'll help you figure out how to test it with SMS.

Reply:
This is working fine for sms. I've tested it.